examples/node-tracing/app.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-env node */
+"use strict";
+const beeline = require("honeycomb-beeline");
+
+const HONEYCOMB_DATASET = `${process.env.HONEYCOMB_DATASET}` || "tracing-example";
```

Comment:
string interpolation results in a literal string "undefined" when there's no env var

```suggestion
const HONEYCOMB_DATASET = process.env.HONEYCOMB_DATASET || "tracing-example";
```